istioctl/cmd/istioctl/convert/cmd.go

```diff
@@ -142,25 +149,57 @@ func convertConfigs(readers []io.Reader, writer io.Writer) error {
 	return nil
 }
 
-func readConfigs(readers []io.Reader) ([]model.Config, error) {
+func readConfigs(readers []io.Reader) ([]model.Config, []*v1beta1.Ingress, error) {
 	out := make([]model.Config, 0)
+	outIngresses := make([]*v1beta1.Ingress, 0)
+
 	for _, reader := range readers {
 		data, err := ioutil.ReadAll(reader)
 		if err != nil {
-			return nil, err
+			return nil, nil, err
 		}
 
 		configs, kinds, err := crd.ParseInputs(string(data))
 		if err != nil {
-			return nil, err
+			return nil, nil, err
+		}
+
+		recognized := 0
+		for _, nonIstio := range kinds {
+			if nonIstio.Kind == "Ingress" &&
+				nonIstio.APIVersion == "extensions/v1beta1" &&
+				nonIstio.ObjectMeta.Annotations["kubernetes.io/ingress.class"] == "istio" {
```

Comment:
I don't think we care about ingress class annotation given this is used in a command line context.